mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/ProfileFragment.java

```diff
@@ -144,6 +144,8 @@ public void showProfile(Client client) {
                 .width((int) getResources().getDimension(R.dimen.user_profile_image_size))
                 .height((int) getResources().getDimension(R.dimen.user_profile_image_size))
                 .endConfig().buildRound(client.getName().substring(0, 1), R.color.colorAccentBlack);
+        ImageView editableImageHint=getView().findViewById(R.id.iv_editable_image_hint);
+        editableImageHint.setVisibility(View.INVISIBLE);
```

Comment:
Nice one, as I expected, but you didn't checked the ./gradlew check requirement. You should do run that command in your android studio terminal, it helps you to check whether your code style is according to the project style but your changes already passed that test on github PR, so no worries, but keep in mind from now that you run "./gradlew check" command before performing commit because many PR's get stuck in checkstyle test and then you have to make a new commit and squash it, so it's better to do it in first attempt. At last just want to say good on I approve it.

Reply:
okay, thanks for the suggestion and review.